### ISSUE TITLE
DVB Subtitles do not work

### ISSUE THREAD

**Issue by JTe-git** [jojo61/vdr-plugin-softhdcuvid#66]
DVB subtitles do not work at all. If you enable DVB subtitles a light transparent filled rectangle box is drawn on the top of the screen when subtitles should be drawn on the bottom. 
If using the plugin vdr-plugin-softhddevice-cuvid DVB subtitles display correctly.
I can provide a test recording if needed.

**Comment by jojo61** [comment]
Sorry for the late answer. Is it still not working ?

**Comment by JTe-git** [comment]
Yep, just tried the latest version and subtitles are not working. They work now fine with the softhddevice though.
